app/src/test/java/ch/epfl/qedit/ExampleUnitTest.java

```diff
@@ -1,17 +0,0 @@
-package ch.epfl.qedit;
-
-import static org.junit.Assert.*;
-
-import org.junit.Test;
-
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() {
-        assertEquals(4, 2 + 2);
```

Comment:
I understand why you deleted this class

Reply:
Yeah, I didn't notice it was there until now, so I thought that it was rather safe deleting it ;).